everyvoice/base_cli/helpers.py

```diff
@@ -76,12 +77,12 @@ def load_config_base_command(
         type[EveryVoiceConfig],
         type[FastSpeech2Config],
         type[HiFiGANConfig],
+        type[TextConfig],
```

Comment:
we shouldn't actually be passing the text config here. better to pass the model config and access the text config from that (i.e. model.text) since we might want to edit other things besides the text config.